docs/framework/get-started/net-core-and-open-source.md

```diff
@@ -26,13 +26,13 @@ This topic provides a brief overview  of what .NET Core is and shows how you can
+-   **Cross-platform:** .NET Core provides key functionality to implement the app features you need and reuse this code regardless of your platform target. It currently supports three main operating systems (OS): Windows, Linux and macOS. You can write apps and libraries that run unmodified across supported operating systems. To see the list of supported operating systems, visit [.NET Core roadmap](https://github.com/dotnet/core/blob/master/roadmap.md).
   
 -   **Open source:** .NET Core is one of the many projects under the stewardship of the [.NET Foundation](http://www.dotnetfoundation.org/) and is available on [GitHub](https://github.com/).  Having .NET Core as an open source project promotes a more transparent development process and promotes an active and engaged community.  
   
--   **Flexible deployment:** there are two main ways to deploy your app: framework-dependent deployment or self-contained deployment. With framework-dependent deployment, only your app and third-party dependencies are installed and your app depends on a system-wide version of .NET Core to be present.  With self-contained deployment, the .NET Core version used to build your application is also deployed along with your app and third-party dependencies and can run side-by-side with other versions.    For more information, see [.NET Core Application Deployment](https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/index).  
-  
--   **Modular:** .NET Core is modular because it is released through NuGet in smaller assembly packages. Rather than one large assembly that contains most of the core functionality, .NET Core is made available as smaller feature-centric packages. This enables a more agile development model for us and allows you to optimize your app to include just the NuGet packages you need. The benefits of a smaller app surface area include tighter security, reduced servicing, improved performance, and decreased costs in a pay-for-what-you-use model.  
+-   **Flexible deployment:** there are two main ways to deploy your app: framework-dependent deployment or self-contained deployment. With framework-dependent deployment, only your app and third-party dependencies are installed and your app depends on a system-wide version of .NET Core to be present.  With self-contained deployment, the .NET Core version used to build your application is also deployed along with your app and third-party dependencies and can run side-by-side with other versions.    For more information, see [.NET Core Application Deployment](../../core/deploying/index.md).
```

Comment:
The casing is fixed on the topic, so for sure it can go to ...

*.NET Core application deployment*